crates/rust-analyzer/src/reload.rs

```diff
@@ -217,6 +226,10 @@ impl GlobalState {
                 .filter_map(Result::ok)
                 .collect();
             let cargo_config = self.config.cargo();
+            let discover_command = self.config.discover_command();
+            let is_quiescent = !(self.discover_workspace_queue.op_in_progress()
+                || self.vfs_progress_config_version < self.vfs_config_version
+                || self.vfs_progress_n_done < self.vfs_progress_n_total);
```

Comment:
this part feels extremely hacky and I don't like it, but its seemingly necessary to avoid spawning another discover command when `fetch_workspaces` is repeatedly called.